### DISCUSSION TITLE
Economy

### DISCUSSION THREAD

**Discussion by armantorkzaban** [tcfev/forDem#140] [General]
# Currency
- Value in forDem is connected with a type of stable crypto currency.

- The coins of this currency are one to one connected to Euro & Dollar.

- Buying and cashing the currency are regulated by the board of forDem @ TCF e.V.

- Cashing and transfering have a fee that generates profit for forDem.

- The profit made will be used solely for moderation purposes.

# Selling Data
- Participation/deliberation/networking and meta data are properties of the users, managed by forDem

- The open source data carries forDem licenses and can be used by governments and civil society actors in return of money

- Any profit will be transferred back to the users who have generated it in forms of ForDem's internal stable coin.

**Comment by desputinski** [comment]
Hm, I don't quite understand if selling any data is ethically okay. I kind of doubt it.... Also we have to discuss the ecologic implications of using crypto currencies.... Let's discuss this in our next meeting!

**Comment by armantorkzaban** [comment]
I would like to have your written arguments here. We can also talk about them, but the written version is the most important as it will be logged and archived in more details.

Please answer the following questions:

- Why should the data generated by the users of the platform be consumed by profit-making companies for free?
Users spend their most valuable resources, their intelligence and time, and generate data. This data although used first by themselves can be also used by other actors. Generation of this data is not free and therefore it must not be offered for free. 

On the other hand the proposal suggests to channel the generated money back to **a)** the users and **b)** the moderation effort
- Why generating money for moderation purposes (one of our heaviest tasks) is not a good idea?
- How generating money for the users could wrong?

---

All crypto currencies are greener than the current banking system and some are by far more ecological than the others. Nothing is holding us from using/developing a stable green crypto currency.

**Comment by desputinski** [comment]
> Why should the data generated by the users of the platform be consumed by profit-making companies for free?

I think the platform is not supposed to be oriented on commercial/monetary values/goals at all. The goal I think is not to generate money but to have an impact on politics and society.

>  Users spend their most valuable resources, their intelligence and time, and generate data. This data although used first by themselves can be also used by other actors. Generation of this data is not free and therefore it must not be offered for free.

From my point of view, it is very critical to sell data to anybody. Who would buy it? Other companies with commercial goals? Or Governments. Do we want that? And what would we gain from that? The question that needs to be asked is: Who would be interested in the data and for which purposes?

About money: I agree that the project needs money. But like other NGOs it is possible to generate money from donations or funds. I understand the need for financial resources, but I think it has to be generated in a legit manner. Acting like a startup, or capitalist market participant, I find very critical.

>  All crypto currencies are greener than the current banking system and some are by far more ecological than the others. Nothing is holding us from using/developing a stable green crypto currency.

Ok, then why not support the crypto currencies, that don't have to be mined with a lot of energy. But I don't know, what we can achieve by supporting it or even creating one our own. Up to this point, I was not able to understand the benefits. Ok, it is decentralized, but on the other hand is highly speculative. Also the motives for creating the myriad of cryptos is mostly commercial. For me not an approach that serves society. Or maybe I just didn't understand the benefits...

PS on Crypto-Mining:
https://www.youtube.com/watch?v=JT22_dUsl5M

**Comment by armantorkzaban** [comment]
> I think the platform is not supposed to be oriented on commercial/monetary values/goals at all. The goal I think is not to generate money but to have an impact on politics and society.

here you are not answering the question at all. The core of the question is this: 
If the generated data is valuable, why should a profit-making organisation be able to use it for free?
or in other words, if the data generated is valuable for anyone else other than the users who generate it, why should this data be given away for free? aren't the users spending here something valuable in order to generate something valuable?

> From my point of view, it is very critical to sell data to anybody. Who would buy it? Other companies with commercial goals? Or Governments. Do we want that? And what would we gain from that? The question that needs to be asked is: Who would be interested in the data and for which purposes?

One example of valuable data would be the data generated from the users before an election or data generated on the way towards a referendum. Many `newspapers` or `interest groups` or governments would like to refer to those data in their publications and their research, and it simply does not belong to them because they have not helped generate it or by using it they will generate a profit that they will or might not share with the creators of that data. 

> About money: I agree that the project needs money. But like other NGOs it is possible to generate money from donations or funds. I understand the need for financial resources, but I think it has to be generated in a legit manner. Acting like a startup, or capitalist market participant, I find very critical.

Here we are not talking about the need. We can talk about that in another conversation dedicated to the topic. The topic here is more general and deals with the value generated by the users of the platform. It is an effort to protect the work of the users.

Protecting the users and giving them the result of their work, and cutting the hands of profit-making institution from the resources belonging to others is legitimate.

> Ok, then why not support the crypto currencies, that don't have to be mined with a lot of energy.

So far we have not defined which currency yet. If we do it, we must go for something stable and green. Let's focus on analysing the (already existing) options. Once we say `stable` it means we do not consider mining and difficulty of solving math equations a source of value.

**Comment by desputinski** [comment]
> If the generated data is valuable, why should a profit-making organisation be able to use it for free?

I think it is very hard to crawl through the whole network and have data of high quality that can be sold. For one thing, it is data about political activities. This has very limited monetary use in comparison to let's say social networks where people unbox consumer products. I honestly don't see a market for a crawling company that is out for the profit...

> One example of valuable data would be the data generated from the users before an election or data generated on the way towards a referendum

Yes, I understand. That could make sense for journalists etc. In the end it is a question, of what kind of culture we want to be part of. Open data, open source or commercial. More Wikipedia or more MS Encarta ;) Or more Google Maps or OpenStreetmap.
If our app is open source it might be more consequent to go down the "open" path...

> Protecting the users and giving them the result of their work, and cutting the hands of profit-making institution from the resources belonging to others is legitimate.

Hm, not sure, if I can follow. My political conviction is more that it is not about profit and should not be. When I use the app to make a social movement stronger and more effective, that is the benefit. 


> So far we have not defined which currency yet....

All right. We can keep that in mind. But we should not fritter and follow too many approaches simultaniously, I think, as long as we are not certain, that it really makes sense to implement something complex like that...

**Comment by armantorkzaban** [comment]
> I think it is very hard to crawl through the whole network and have data of high quality[...]

not really. Core of our job is to facilitate exactly that.

> For one thing, it is data about political activities. This has very limited monetary use [...]

you are underestimating that. Just research what amount of money is spent in political and media campaigns 

> Yes, I understand. That could make sense for journalists etc. In the end it is a question, of what kind of culture we want to be part of. Open data, open source or commercial. More Wikipedia or more MS Encarta ;) Or more Google Maps or OpenStreetmap.
If our app is open source it might be more consequent to go down the "open" path...

Our open source does not necessarily mean that **profit-making entities** should be allowed to use the generated data and make money with it by keeping the public away from the results through paywalls or by promoting modern slavery.
Answer this question please: why should such companies be allowed to use open source code and the generated data based on that?

> Hm, not sure, if I can follow. My political conviction is more that it is not about profit and should not be. When I use the app to make a social movement stronger and more effective, that is the benefit.

There are other factors at play here and not just politics. There are economic, civil and social activities involved. Here we have producers (users and public that spend their resources to generate data and cause action), means of production (partly the platform which has to be moderated), [sociopolitical actions ...], and value-added products (partly the data; all sorts of it). 
Why would some actors (here the ones that slow down or prevent the very sociopolitical changes that are aimed for by the users) make money from the products?

> I think, as long as we are not certain, that it really makes sense to implement something complex like that...

We can involve experts here and analyse the complexity. Definitely it is not going to be in the MVP and it is not planned for the V.1

**Comment by armantorkzaban** [comment]
@desputinski, let us continue over here in a discussion instead of an issue.

**Comment by armantorkzaban** [comment]
related topic: https://github.com/tcfev/task-force-nika/issues/61

**Comment by IranNewsWatch** [comment]
I think promoting fund from the data needs to be in the terms and conditions and I think that would look aweful on us. Actually the opposit, I suggest copy-righting user's generated content (or some of them, e.g. articles) to limit the comercial use, and let the regulations do the work; use for the analysis of the political environment or helping people of Iran would be free (which makes sense to be).

As I sais before, first we need to deliver something trustworthy then we can see if we have to squeeze something like data to make money out of it.

Also I don't think (anonymous) political discussions are much worthy for businesses. I think it might be useful to think tanks and politicians. And that's actually our goal, to be heard, freely!

**Comment by armantorkzaban** [comment]
From another convo: 
our data is open and available to everyone who needs them. However, here I am proposing that the for-profit entities using this data must pay and that is because they are making profit off the people's data. Moreover we do not pocket the hypothetical money. We give it to the ones who generated it: the people on our platform, who generated the data in first place.
This means e.g that the results of a large survey can only then be used by NYT when they pay for it and when they did, the people who have voted in the survey will get paid.
We want to make sure that no one makes profit with people's data (what is publicly available) and if they did, they will pay those who have generated it.
It is basically what you are talking about in your proposal @IranNewsWatch in [Helium](https://github.com/tcfev/task-force-nika/issues/61)'s topic. You say we incentivize good use of the platform (quality work, help in extension of the network) by paying for that, IOW we enjoy (having fun has costs) therefore we pay for it; So should do the news agency who reports on a survey done over forDem too.

**Comment by ghost** [comment]
Let me chip in here as someone living in Iran and a potential user of this to-be platform.

I don't know exactly what problem(s) this platform is supposed to solve or how it's going to be used to help the Mahsa Movement. But as we're getting closer to the fifth month of this movement and people getting more tired every day, you are discussing whether to sell or not sell some data that doesn't even exist yet.

Just a month ago, half the city was closed for 3 days in a row without any organization or expectation of rewards, because of a general consensus born out of social media. Our office with 50 employees was closed along with many other offices/stores/etc. Closing the workplace costs money. Iranians are not waiting for a platform to vote and get paid. They are waiting for a plan to unite and succeed in their revolution, not only they don't expect payment, they are willing to spend.

Also I don't care if NYT makes money out of my data as long as they create more exposure. Actually I don't care at all even if they don't make exposure. I just want to have a free country and I think spending time on making sure others don't make money is kinda wasteful right now.

If success of this movement is the main goal then this discussion is pointless. We don't really have much time left, people are getting tired and killed. If you have other goals then I apologize for being a bit harsh.

**Reply by armantorkzaban** [reply]
Legit points. We are just sporadically discussing this issue and it is not at all our main focus. It was raised at some point in time and we decided to document it. I would say it is the least important part of our work atm and we can postpone the discussion even longer.
But hey, why not joining us in the production of what is needed? We have a team here ready and active, join and reprioritise, accelerate the work and help us put the emphasis where it must be.

**Reply by ghost** [reply]
I think forDem could be useful but as I am living in the middle of a revolution that could fail at any moment, I really don't have the time to build a full digital democracy platform, whatever that is. If you are willing to change your focus to just helping the Iranian people unite and postpone everything that takes a lot of effort to build and does not immediately help the Iran revolution, then sure, I would love to join.

**Reply by ghost** [reply]
Perhaps I should give you at least one example of what I mean. Consider the "voting" feature.

The way I understood it by looking around in your issues and discussions, you are trying to implement "digital elections". This is a very hard problem which is currently unsolved and as I said, I don't have much free time to try to solve a problem that people much smarter than me couldn't solve. Even if you have found a way to implement digital voting that is anonymous and fair (every real life Iranian can vote but only once, everyone else cannot vote), you first need to publish a paper and have a proof-of-concept implementation, get it checked and vetted by security experts, educate people that it actually is safe and can be trusted (this is very hard to do even if the system is actually anonymous and fair, even Bitcoin hasn't been able to do yet), etc. This is gonna take too much time that I don't have.

If by "voting" you mean anything other than "digital elections", then I don't see how that could help the revolution in its current state. Something that is a little bit better than twitter voting or google forms is not helpful right now (and I highly doubt we can even match them, let alone build something better).

**Reply by armantorkzaban** [reply]
Hello @asnadazadi,

It is very difficult to help people in Iran (in this revolution and in a meaningful way) by only offering them a digital election software. If that is what you are trying to say, you are right, and I can only agree with you.

But some of the points that you mentioned are wrong and not accurate enough in the case of forDem, and its parent organisation, the Transnational Community Federation e.V. (TCF).

I will explain below why. 
Although a) [participation features](https://github.com/tcfev/forDem/issues?q=is%3Aopen+label%3AParticipation+sort%3Aupdated-desc) are only parts of the features that the platform aims to offer,
b) there are ways to mitigate or eliminate the effect of rogue actors on survey and elections,
c) up until today there has not been a thing that has stopped us from innovation & development according to standards,
these have never been our only goals.
 
- The first thing that you should know is that we have started forDem project not three months ago, but almost 6 years ago.
 
- From the day zero we anticipated total internet shutdown imposed on people of Iran by the islamic state, and because of just that we kept networking functionalities on radio layer (WiFi-Direct, LTE D2D, etc.) one of our top priorities.
Today, as I am writing this to you, there is an ongoing international effort with the sole aim to bring mesh for mid-size networks (250 nodes, full connectivity) closer to reality, and you guessed it right, the result will be a integrated into forDem.

But even these are not all we are doing right now. We know the problem is big, and we know that a systemic solution is what's needed, that's why we have helped create a federation of communities, in which we are just one member.

The communities we are a part of develop and run various services & products dedicated to the Iranian revolutionaries ranging from delivering Starlinks and e-democracy tools to hosting mindful yoga & community awareness classes, they are involved in scientific research projects in fields of education (university in exile), LGBTIQ+ rights (NQIfFM), support refugees in countries where they have little to no rights, and coordinate various IT & digital democracy efforts.

I see no reason why we should not join the efforts, but that would not have meant much without you actually knowing us.
